apps/aevm/test/aevm_dummy_chain.erl

```diff
@@ -8,18 +8,22 @@
 
 -module(aevm_dummy_chain).
 
--behaviour(aec_vm_chain_api).
+%% -behaviour(aec_vm_chain_api).
```

Comment:
Why no more `-behaviour`?

Reply:
It can not find behavior if you run just one test suite